src/Components/Inputs/Input.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { WrapperInput, Wrapper } from './styled';
+
+const Input = ({
+  type, placeholder, width, height,
+}) => (
+  <Wrapper width={width} height={height}>
```

Comment:
remove redundant wrapper, pass width and height straight to input

Reply:
fixed and create InputWrapper with height and width;